src/components/ActionFetch.vue

```diff
@@ -136,4 +138,31 @@ export default {
   margin-top: -1.7px;
   margin-left: 2px;
 }
+
+/* TOOLTIP CAP COVER IMAGE */
+
+.xivtooltip-thumbnail::before{
+  content: '';
+  background: url('../src/assets/cover.png');
```

Comment:
This isn't going to work, the src and root directories aren't in the distributed bundle.

Reply:
this will require editing the rollup configuration before this can work. I'll PR it tomorrow after I get home. After that, the line will likely need to change from:

```css
background: url('../src/assets/cover.png');
```
to
```css
background: inline('cover.png')
```

That's assuming we want to leverage more postcss features. Will need to think about it while I'm out.
If we do this, then it's worthwhile to centralize the css somewhere and import them somewhere (like we're doing with popper's base css when we bundle it). Another reason to centralize it will be because changing all three references will result in 3x the delta increase (since it's going to inline the same png image 3 times)

Reply:
While I was making them (and couldn't find the right syntax for the one in src) I just loaded the png on my site and used it i.e., https://saltedxiv.com/cover.png Is external hosting it like that/but somewhere else an option you'd want to consider?

Reply:
One thought is remaking it in CSS that way it's just a part of the code. Probably a lot lighter than including a png in the bundle. Not sure how best to go about that though.

Reply:
If we are going to host it should use this one since its like 1kb instead of 16kb. Gotta optimize that shizz for web yo.
![cover (1)](https://user-images.githubusercontent.com/39720490/99321791-a36e5d80-283c-11eb-9166-3b52ce94b291.png)

Reply:
I'd def base64 encode and inline it, no other solution makes sense for a library.